src/build.c

```diff
@@ -1081,8 +1082,15 @@ static gchar *build_create_shellscript(const gchar *working_dir, const gchar *cm
 #ifdef G_OS_WIN32
 	/* Expand environment variables like %blah%. */
 	expanded_cmd = win32_expand_environment_variables(cmd);
-	str = g_strdup_printf("cd \"%s\"\n\n%s\n\n%s\ndel \"%%0\"\n\npause\n", working_dir, expanded_cmd, (autoclose) ? "" : "pause");
+	str = g_strdup_printf(
+		"cd \"%s\"\n\n\n%s\n\n%s\ndel \"%%0\"\n\npause\n",
+		working_dir, expanded_cmd, (autoclose) ? "" : "pause");
 	g_free(expanded_cmd);
+	/* Convert script content into system codepage */
+	utf8_script_content = win32_convert_to_system_codepage(str, error);
```

Comment:
@codebrainz 1) I'm talking about `run_cmd` not `term_cmd` and 2) as I have pointed out elsewhere `utils_get_locale_from_utf8()` does nothing on windows and 3) the command is sent to `spawn_*` not a `g_*` function, and 4) most of our problems appear to be that `spawn_*` uses windows calls and not glib calls and do not convert from the glib rule of `always UTF8` to the real windows rule of windows locale.